3.3/examples/django/cleanup.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+echo "-> removing openshift.local.* directories..."
+sudo rm -rf openshift.local.*
+
+echo "-> stopping all k8s docker containers..."
+docker ps | awk '{ print $NF " " $1 }' | grep ^k8s_ | awk '{print $2}' |  xargs -l -r docker stop
```

Comment:
Since this is part of the maintained image, I'd rather have it up to date more or less and this file isn't required here, the same as with bootstrap.sh you had previously.